app/src/main/java/com/wearerommies/roomie/presentation/ui/login/LoginRoute.kt

```diff
@@ -0,0 +1,136 @@
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.LocalLifecycleOwner
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import androidx.lifecycle.flowWithLifecycle
+import com.wearerommies.roomie.R
+import com.wearerommies.roomie.presentation.core.extension.noRippleClickable
+import com.wearerommies.roomie.ui.theme.RoomieAndroidTheme
+import com.wearerommies.roomie.ui.theme.RoomieTheme
+
+@Composable
+fun LoginRoute(
+    paddingValues: PaddingValues,
+    navigateUp: () -> Unit,
+    navigateToHome: () -> Unit,
+    viewModel: LoginViewModel = hiltViewModel()
+) {
+    val context = LocalContext.current
+    val lifecycleOwner = LocalLifecycleOwner.current
+    val state by viewModel.state.collectAsStateWithLifecycle()
+
+    LaunchedEffect(viewModel.sideEffect, lifecycleOwner) {
+        viewModel.sideEffect.flowWithLifecycle(lifecycleOwner.lifecycle)
+            .collect { sideEffect ->
+                when (sideEffect) {
+                    is LoginSideEffect.StartLogin -> {
+                        // 필요 시 StartLogin 추가 처리
+                    }
+
+                    is LoginSideEffect.LoginSuccess -> {
+                        navigateToHome()
+                    }
+
+                    is LoginSideEffect.LoginError -> {
+                        // 필요 시 LoginError 추가 처리
+                    }
+                }
+            }
+    }
+
+    LoginScreen(
+        paddingValues = paddingValues,
+        navigateUp = navigateUp,
+        state = state,
+        onLoginClick = { viewModel.startKakaoLogin(context) },
+    )
+
+}
+
+@Composable
+fun LoginScreen(
+    paddingValues: PaddingValues,
+    navigateUp: () -> Unit,
+    state: LoginState,
+    onLoginClick: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    Column(
+        modifier = modifier
+            .fillMaxSize()
+            .background(RoomieTheme.colors.grayScale1)
+            .padding(paddingValues),
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+        Icon(
+            modifier = Modifier
+                .padding(top = 105.dp),
+            imageVector = ImageVector.vectorResource(R.drawable.ic_roomie_logo),
+            contentDescription = stringResource(R.string.roomie_logo),
+            tint = Color.Unspecified
+        )
+        Text(
+            modifier = Modifier
+                .padding(top = 20.dp),
+            text = stringResource(R.string.login_title),
+            style = RoomieTheme.typography.title2Sb16,
+            color = RoomieTheme.colors.gradientPrimary
+        )
+
+        //todo: image 추가 예정
+
+        Spacer(
+            modifier = Modifier
+                .weight(1f)
+        )
+
+        Image(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(start = 20.dp, end = 20.dp, bottom = 42.dp)
+                .noRippleClickable { onLoginClick() },
```

Comment:
람다식을 사용하게 되면 리컴포지션이 일어날 수 있어요! 기존 람다 객체를 그대로 전달합시당!
```suggestion
                .noRippleClickable(clickable = onLoginClick)
```